analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantConditionalAroundAssignment.CSharp9.cs

```diff
@@ -3,12 +3,49 @@
 
```

Comment:
Please try to cover or at least document the following FNs:
```
object a = null;

switch (a)
{
    case null:
        {
            a = null;
        }
        break;
    default:
        break;
}

if ((x,y) is (1,2))
{
    x = 1;
    y = 2;
}
```

Reply:
Documented them as FNs for now